changelog/fix-deprecation-notice-on-reports-screen

```diff
@@ -1,4 +0,0 @@
-Significance: patch
-Type: fixed
-
-Deprecation notice on the reports screen when using PHP 8.1
```

Comment:
Having one changelog for the deprecation notices might be better because there are so many.